pkg/kubelet/status/generate_test.go

```diff
@@ -141,3 +141,113 @@ func getNotReadyStatus(cName string) v1.ContainerStatus {
 		Ready: false,
 	}
 }
+
+func TestGeneratePodInitializedCondition(t *testing.T) {
```

Comment:
Can you please move this right under https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/status/generate_test.go#L117 ?

This can make sure the layout of the file is:

```
Test function
Test function
helper function
helper function
helper function
...
```